### PR TITLE
Chore: Stories for Select performance

### PR DESCRIPTION
**Pull request by joshhunt**
**What is this feature?**

Adds an (internal) storybook story for testing React/react-select performance

**Which issue(s) does this PR fix?**:

Related to https://github.com/grafana/grafana/issues/57675

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/grafana-ui/src/components/Select/SelectPerf.story.internal.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import { Label } from '../Forms/Label';
+
+import { Select, VirtualizedSelect } from './Select';
+
+const meta: Meta = {
+  title: 'Forms/Select (Perf)',
+  decorators: [withCenteredStory, withHorizontallyCenteredStory],
+  argTypes: {
+    numberOfOptions: {
+      defaultValue: 10_000,
```
Comment (ashharrison90) [inline packages/grafana-ui/src/components/Select/SelectPerf.story.internal.tsx:17]: is this some syntax trick i didn't know, or is just working by accident? 😱

**Review by ashharrison90 — APPROVED** [review]
good mob 🔱

File: packages/grafana-ui/src/components/Select/SelectPerf.story.internal.tsx (hunk 1 above)
Comment (joshhunt) [reply packages/grafana-ui/src/components/Select/SelectPerf.story.internal.tsx:17]: [Numeric separators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#numeric_separators)

Comment (ashharrison90) [reply packages/grafana-ui/src/components/Select/SelectPerf.story.internal.tsx:17]: TIL

**Pull request merged**